### PR TITLE
Rename module_name and module_type

### PR DESCRIPTION
**Pull request by ddurieux**
Rename:

* __module_name__ in __module_alias__
* __module_type__ in __module__

### CODE REVIEW

File: alignak/objects/config.py [hunk 1]
```diff
@@ -1749,38 +1749,38 @@ def fill_default_satellites(self):
...
-        :type mod_type: str
+        :param module: module type to search
+        :type module: str
         :return: True if mod_type is found else False
         :rtype: bool
         """
         for broker in self.brokers:
             for module in broker.modules:
-                if hasattr(module, 'module_type') and module.module_type == mod_type:
+                if hasattr(module, 'module') and module.module == module:
```
Comment (Seb-Solon) [inline alignak/objects/config.py]: module.module is strange but still legit in python

File: alignak/objects/config.py [hunk 2]
```diff
@@ -1749,38 +1749,38 @@ def fill_default_satellites(self):
...
+        :param module: module type to search
+        :type module: str
         :return: True if mod_type is found else False
         :rtype: bool
         TODO: Factorize it with got_broker_module_type_defined
         """
         for scheduler in self.schedulers:
             for module in scheduler.modules:
-                if hasattr(module, 'module_type') and module.module_type == mod_type:
+                if hasattr(module, 'module') and module.module == module:
```
Comment (Seb-Solon) [inline alignak/objects/config.py]: Same here

File: alignak/objects/config.py (hunk 1 above)
Comment (ddurieux) [inline alignak/objects/config.py]: it's the 'name' of module to import, have you another string name for this? I haven't idea of a good name for this

Comment (ddurieux) [inline alignak/objects/config.py]: New proposition of name: python_name

**Pull request merged**